c2corg_api/views/document_changes.py

```diff
@@ -61,7 +61,7 @@ def get(self):
 def get_changes_of_feed(token_id, limit, user_id=None):
     query = DBSession. \
         query(HistoryMetaData.id, HistoryMetaData.user_id). \
-        order_by(HistoryMetaData.written_at.desc())
+        order_by(HistoryMetaData.id.desc(), HistoryMetaData.written_at.desc())
```

Comment:
Do you need to order by the two columns? Wouldn't `HistoryMetaData.id` be enough?